SU-HSDO/suhumsci
# feat(STN-201):  qa feedback and bug fixes

# READY FOR REVIEW

## Summary
Please see [JIRA card](https://sparkbox.atlassian.net/browse/STN-201) for different bugs this addresses. I tried to break up each into issue into a different commit for ease of review.

- it might also be a good idea for me to zoom and go through this with Alex on monday to make sure I addressed his concerns he wrote up

## Steps to Test
1. Make sure the bugs appear to be fixed that are listed in the QA JIra card
2. Ensure that when logged in / out, disabling js / enabling js that you see the correct class on the `html` element.
3. run `npm run test` and make sure the tests pass

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
